enhancements/network/on-prem-configurable-lb-dns-stack.md

```diff
@@ -0,0 +1,200 @@
+---
+title: on-prem-configurable-lb-dns-stack
+authors:
+  - "@yboaron"
+reviewers:
+  - TBD
```

Comment:
this must be reviewed by API & Auth team (@sttts or @deads2k) as LB is on their critical path